backend/src/main/java/com/daedan/festabook/announcement/domain/Announcement.java

```diff
@@ -0,0 +1,40 @@
+package com.daedan.festabook.announcement.domain;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EntityListeners;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.time.LocalDateTime;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Entity
+@Getter
+@EntityListeners(AuditingEntityListener.class)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Announcement {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column(nullable = false)
+    private String content;
+
+    @CreatedDate
+    @Column(nullable = false)
+    private LocalDateTime createdAt;
+
+    public Announcement(String title, String content) {
+        this.title = title;
+        this.content = content;
+    }
```

Comment:
생성자 매개변수 줄바꿈 컨벤션에 맞게 변경해주세요.

```suggestion
    public Announcement(
        String title, 
        String content
    ) {
        this.title = title;
        this.content = content;
    }
```